Makefile

```diff
@@ -73,6 +73,7 @@ str_match_test:
 # testing
 ##
 test: joy joy_test.py
+	./download_test_pcaps.sh test_pcaps/pcapr
```

Comment:
I think the idea is good, but the shell script also won't run on Windows which we support now. Python may be a better choice for the script. @bhudson33 What are your thoughts on this?